Sources/JWT/Signers/HMACSigner.swift

```diff
@@ -1,43 +1,57 @@
 import Core
 import HMAC
 
-public struct HS256: HMACSigner {
+public class HS256: HMACSigner {
     public let key: Bytes
     public let method = HMAC.Method.sha256
 
-    public init(key: Bytes) {
+    public required init(key: Bytes) {
```

Comment:
Oops, the `required` was accidental carry over from the `Key` protocol. I think making these `final` is a good idea.